python/lsst/ip/isr/isrTask.py

```diff
@@ -721,6 +721,17 @@ class IsrTaskConfig(pipeBase.PipelineTaskConfig,
         default=True,
     )
 
+    # Amp offset correction.
+    doAmpOffset = pexConfig.Field(
```

Comment:
This and below might be renamed, but I'm actually satisfied with this.  But we should ask more broadly.  I just want to make sure that a search of the codebase will come up with something consistent and relatively unique.  SEO style!

Reply:
This name has been preserved.